user/security-in-qubes/ctap-proxy.md

```diff
@@ -0,0 +1,144 @@
+---
+lang: en
+layout: doc
+permalink: /doc/ctap-proxy/
+redirect_from: /doc/u2f-proxy/
+ref: 167
```

Comment:
I was told that new doc pages should not include a `ref:` line, because there is a script responsible for adding an appropriate value based on all other pages.

Reply:
Oh, wait, I guess this is not actually a new doc page, but rather a replacement for the existing u2f page. If that's the case, never mind.